### PR TITLE
Feature/transition page

### PR DESCRIPTION
**Pull request by Justicesvr**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [x] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [x] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [x] My `Javascript` generate no new console errors
- [x] I tested my code cross browsers
- [x] My slice is pixel perfect for both desktop and mobile according to design
- [x] I conducted basic QA to assure all features are working
- [x] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by jaalazard — DISMISSED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by jaalazard — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]

**Review by jaalazard — DISMISSED** [review]

**Review by No3ml3 — DISMISSED** [review]

File: packages/frontend/web/src/pages/Login.tsx [hunk 1]
```diff
@@ -79,6 +87,11 @@ export default function Login() {
 
   return (
     <div className='bg-pastel-blue flex h-screen items-center justify-center p-5'>
+      {!!isLoading && (
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/Login.tsx]: Pourquoi `!!` ?

File: packages/frontend/web/src/pages/AddCocktail.tsx [hunk 2]
```diff
@@ -3,20 +3,28 @@ import { FormProvider, useForm } from 'react-hook-form';
...
 
 export default function AddCocktail() {
   const methods = useForm<CocktailForm>();
   const [isModalShown, setIsModalShown] = useState(false);
   const [actualIngredient, setActualIngredient] = useState(0);
+  const [isShake, setIsShake] = useState(false);
 
   return (
     <FormProvider {...methods}>
+      {!!isShake && (
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/AddCocktail.tsx]: ?

File: packages/frontend/web/src/components/form-cocktail/SoftDrinks.tsx [hunk 3]
```diff
@@ -35,16 +35,17 @@ export default function SoftDrinks({
...
           checked={withAlcohol}
           onChange={() => {
             setWithAlcohol(!withAlcohol);
           }}
         />
-        <div className="peer h-6 w-11 rounded-full bg-gray-200 after:absolute after:start-[2px] after:top-[2px] after:h-5 after:w-5 after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-[''] peer-checked:bg-blue-600 peer-checked:after:translate-x-full peer-checked:after:border-white peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-blue-300 rtl:peer-checked:after:-translate-x-full dark:border-gray-600 dark:bg-gray-700 dark:peer-focus:ring-blue-800" />
+        <div className="peer h-6 w-11 rounded-full bg-gray-700  after:absolute after:start-[2px] after:top-[2px] after:h-5 after:w-5 after:rounded-full after:border after:border-gray-700  after:bg-white after:transition-all after:content-[''] peer-checked:bg-transparent peer-checked:after:translate-x-full peer-checked:after:border-white peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-transparent rtl:peer-checked:after:-translate-x-full dark:border-gray-600 dark:bg-gray-700 dark:peer-focus:ring-gray-700" />
```
Comment (YDrogen) [inline packages/frontend/web/src/components/form-cocktail/SoftDrinks.tsx]: double espaces x2

File: packages/frontend/web/src/pages/Community.tsx [hunk 4]
```diff
@@ -31,73 +31,79 @@ export default function Community() {
             <img
               src='/home/home-1.png'
               alt='Girls talking'
-              className='absolute top-2 z-20 h-[13rem] sm:h-[11rem] sm:w-[11rem]'
+              className='absolute top-2 z-20  h-[11rem] w-[11rem]'
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/Community.tsx]: idem

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/frontend/web/src/pages/Login.tsx (hunk 1 above)
Comment (Justicesvr) [reply packages/frontend/web/src/pages/Login.tsx]: je ne sais pas

File: packages/frontend/web/src/pages/AddCocktail.tsx (hunk 2 above)
Comment (Justicesvr) [reply packages/frontend/web/src/pages/AddCocktail.tsx]: mais ca marche

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]

File: packages/frontend/web/src/pages/Login.tsx (hunk 1 above)
Comment (YDrogen) [reply packages/frontend/web/src/pages/Login.tsx]: Ça load ou ça load pas du coup ? T'as inversé ta condition là, c'est normal ?

File: packages/frontend/web/src/pages/AddCocktail.tsx (hunk 2 above)
Comment (YDrogen) [reply packages/frontend/web/src/pages/AddCocktail.tsx]: idem

**Review by No3ml3 — APPROVED** [review]

**Review by jaalazard — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**